tests/integrations/python/tests/test_google.py

```
@@ -3024,6 +3070,215 @@ def test_43_google_search_grounding_streaming(self, test_config):
+            config=types.GenerateVideosConfig(aspect_ratio="9:16"),
+        )
+
+        completed = _poll_video_operation(client, operation)
+        generated_video = _extract_first_generated_video(completed)
+
+        assert getattr(completed, "done", False) is True
+        assert generated_video is not None
+
+    @skip_if_no_api_key("gemini")
+    @pytest.mark.timeout(1200)
+    def test_44c_gemini_video_resolution(self, test_config):
```

Ruff (0.15.1)

[warning] Unused method argument: `test_config`

(ARG002)